love/components/lover-about.tsx

```diff
@@ -99,7 +99,17 @@ function Seeking(props: { lover: Lover }) {
     asSentence: true,
     capitalizeFirstLetterOption: false,
   })
+  if (prefGender.length == 5) {
+    seekingGenderText = "Interested in anyone"
+  }
   const ageRangeText = `between ${min} - ${max} years old`
+  if (min == 18 && max == 99) {
+    ageRangeText = "of any age"
+  } else if (min == 18) {
+    ageRangeText = `younger than ${max}`
```

Comment:
Not the best idea for min = 18, max = 18 since that will show "younger than 18" carrying wrong implications.

Reply:
should really be `younger than ${max + 1}`, a very subtle distinction that makes all the difference